### PR TITLE
feat: Get and save clients mlsPublicKeys (WPB-2308)

### PR DESCRIPTION
**Pull request by borichellow**
# What's new in this PR?

Added `mls_public_keys` field into `Client` models and DB. It's needed to display MLS info in settings.

Needs releases with:

Could be merged without any problems for the AR.

### CODE REVIEW

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/ClientRepository.kt [hunk 1]
```diff
@@ -159,7 +159,10 @@ class ClientDataSource(
         return wrapApiRequest { clientApi.fetchSelfUserClient() }
             .onSuccess { clientList ->
                 val selfUserIdDTO = selfUserID.toApi()
-                val list = clientList.map { clientMapper.toInsertClientParam(it, selfUserIdDTO) }
+                val list = clientList.map {
+                    println("cyka ${it.model} ${it.mlsPublicKeys}")
```
Comment (typfel) [inline logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/ClientRepository.kt:163]: Delete?

```suggestion
```

**Review by typfel — CHANGES_REQUESTED** [review]
Some minor comments otherwise looks fine but can you also add a description to the PR which explains why we need to save the public keys?

**Comment by borichellow** [comment]
> Some minor comments otherwise looks fine but can you also add a description to the PR which explains why we need to save the public keys?

it was kinda draft, now everything is fixed :)

**Review by typfel — APPROVED** [review]

**Review by alexandreferris — APPROVED** [review]

**Review by mchenani — APPROVED** [review]
🥳

**Pull request merged**